libs/dyn/value.go

```diff
@@ -52,6 +52,24 @@ func NewValue(v any, loc []Location) Value {
 	}
 }
 
+func NewValueAnchor(v any, loc []Location) Value {
```

Comment:
You could use `NewValue(...).MarkAnchor()` to avoid duplicating the impl.

Reply:
Right, done in bef5814724e666dc0fcd0d704eca1b4aad8a0953